CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 Not yet released; provisionally v2.0.0 (may change).
 
+### GetMapLeafByRevision API
+New API for fetching a single map leaf separates batch APIs from single shot
```

Comment:
I think "single shot" is open to being misunderstood. Maybe just remove the last bit and note batch API still exists.